util/src/main/java/com/google/cloud/hadoop/util/HttpTransportFactory.java

```diff
@@ -23,29 +23,35 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.flogger.GoogleLogger;
 import java.io.IOException;
+import java.io.InputStream;
 import java.net.Authenticator;
+import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.PasswordAuthentication;
 import java.net.Proxy;
+import java.net.Socket;
+import java.net.SocketException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.security.GeneralSecurityException;
 import javax.annotation.Nullable;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLSocketFactory;
 
 /** Factory for creating HttpTransport types. */
 public class HttpTransportFactory {
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
 
   /**
-   * Create an {@link HttpTransport} based on an type class.
+   * Create an {@link HttpTransport} with socketKeepAlive true
    *
    * @return The resulting HttpTransport.
    * @throws IllegalArgumentException If the proxy address is invalid.
    * @throws IOException If there is an issue connecting to Google's Certification server.
    */
   public static HttpTransport createHttpTransport() throws IOException {
     return createHttpTransport(
-        /* proxyAddress= */ null, /* proxyUsername= */ null, /* proxyPassword= */ null);
+        /* proxyAddress= */ null, /* proxyUsername= */ null, /* proxyPassword= */ null, true);
```

Comment:
Please follow the established pattern of documenting the named parameters: `/* socketKeepAlive= */ true`.

Reply:
Not needed anymore.